include/grpc/grpc_security.h

```diff
@@ -240,6 +240,24 @@ GRPCAPI grpc_channel_credentials* grpc_ssl_credentials_create(
     const char* pem_root_certs, grpc_ssl_pem_key_cert_pair* pem_key_cert_pair,
     const verify_peer_options* verify_options, void* reserved);
 
+/** Sets the minimum TLS version that will be negotiated during the TLS
```

Comment:
We should clarify in the comment that min and max TLS version only work for OpenSSL and BoringSSL 1.1 and above.